pymeasure/instruments/anritsu/anritsuMS2090A.py

```diff
@@ -95,21 +95,21 @@ def __init__(self, adapter, name="Anritsu MS2090A Handheld Spectrum Analyzer", *
     frequency_center = Instrument.control(
         "FREQuency:CENTer?", "FREQuency:CENTer %g",
         "Sets the center frequency in Hz",
-        validator=truncated_range,
-        values=[-99999999995, 299999999995]
+        validator=strict_range,
+        values=[9E3, 54E9]

```

Comment:
Did you correct an implementation error, because the limits changed.

Reply:
I change the limits as [LongnoseRob](https://github.com/LongnoseRob) says me. In the manual appears the limits that I write at the begin

Reply:
Yes, but If you read [Programming Manual](https://dl.cdn-anritsu.com/en-us/test-measurement/files/Manuals/Programming-Manual/10580-00445R.pdf) of the instrument, the limits are other. But I think that It is better your recommendation.